src/server/routes/proposals.ts

```diff
@@ -35,7 +36,22 @@ proposals.get('/', async (req: Request, res: Response) => {
 })
 
 proposals.post('/', async (req: Request, res: Response) => {
-  const { boardName: name, topicId, payload: newProposal }: fetchBody = req.body
+  const {
+    boardName: name,
+    topicId,
+    payload: { description },
```

Comment:
I don't understand your code fully, but payload sounds like a very generic naming, you will know it the best, it was just something i've noticed.